_includes/hub/header.html

```diff
@@ -1,4 +1,4 @@
-<div class="siteHeader siteHeader--hub">
+<div class="siteHeader siteHeader--hub"><h1 class="-screenReader" id="search">{{ page.title }}</h1>
```

Comment:
Hi @israelcefrin, can you move the `h1` to the next line for better readability/formatting?

And should it have `id="search"` or was that copied unintentionally?